### PR TITLE
For Ember Times No. 109: EmberCamp Chicago Writeup

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-08-02-the-ember-times-issue-109.md [hunk 1]
```diff
@@ -24,86 +25,90 @@ The high level goals for 2019-2020 are:
...
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [Get Ready for EmberCamp Chicago 🍕](http://embercamp.com/)
+
+This year's edition of [EmberCamp Chicago](http://embercamp.com/) is taking place on **September 16th**, and you should **not miss it**!
+
+You can already check out the [full talk schedule](http://embercamp.com/speakers) which includes insights into Ember and modern best practices presented by [Jen Weber](https://twitter.com/jwwweber), [Edward Faulkner](https://twitter.com/eaf4), [Olivia Liddell](https://twitter.com/oliravi), [Dan Gebhardt](https://twitter.com/dgeb), and many other amazing speakers ✨
```
Comment (ijlee2) [inline source/2019-08-02-the-ember-times-issue-109.md]:
```suggestion
You can already check out the [full talk schedule](http://embercamp.com/speakers), which includes insights into Ember and modern best practices presented by [Jen Weber](https://twitter.com/jwwweber), [Edward Faulkner](https://twitter.com/eaf4), [Olivia Liddell](https://twitter.com/oliravi), [Dan Gebhardt](https://twitter.com/dgeb), and many other amazing speakers! ✨
```

File: source/2019-08-02-the-ember-times-issue-109.md [hunk 2]
```diff
@@ -24,86 +25,90 @@ The high level goals for 2019-2020 are:
...
+
+[Get your ticket today](https://www.eventbrite.com/e/embercamp-chicago-2019-registration-63595006376) before they run out! And If you or your company would be interested in being a sponsor of EmberCamp Chicago, please reach out via [embercamp.chicago@gmail.com](mailto:embercamp.chicago@gmail.com).
```
Comment (ijlee2) [inline source/2019-08-02-the-ember-times-issue-109.md]:
```suggestion
[Get your ticket today](https://www.eventbrite.com/e/embercamp-chicago-2019-registration-63595006376) before they run out! If you or your company would be interested in being a sponsor of EmberCamp Chicago, please reach out via [embercamp.chicago@gmail.com](mailto:embercamp.chicago@gmail.com).
```

**Review by ijlee2 — APPROVED** [review]
Looks great, thanks for covering EmberCamp (look forward to your talk as well)!

**Comment by jayjayjpg** [comment]
@ijlee2 Thank you 🌈

**Pull request merged**